src/vm/runners/cairo_runner.rs

```diff
@@ -2844,6 +1678,7 @@ mod tests {
                 .segments
                 .add(&mut cairo_runner.vm.memory, None);
         }
+        // Memory initialization without macro
```

Comment:
Why did this have to be done without a macro?

Reply:
The test fails when the memory isn't initialized this way, I believe it has something to do with the memory gaps. I tried using the macro for all values but the last one and it also didn't work